docs/using/plugins.md

```diff
@@ -1,65 +1,88 @@
 
-Recall that the high-level objectives of parameter handlers in the toolkit are to
+Currently, systems using custom interactions can only be exported to OpenMM. There are two routes for this export: one using existing Interchange machinery and another that allows for completely custom behavior.
 
-* store force field parameters
-* run SMIRKS-based typing
-* enable construction and modification via the Python API
-* enable serialization and deserialization
+[`ParameterHandler`]: openff.toolkit.typing.engines.smirnoff.parameters.ParameterHandler
+[`SMIRNOFFCollection`]: openff.interchange.plugins.SMIRNOFFCollection
+[SetupTools entry points]: setuptools:userguide/entry_point
+[`openmm.Force`]: openmm.openmm.Force
 
-and the objectives of the corresponding collections in Interchange are to
+## Creating a custom `ParameterHandler`
 
-* store "system-level" parameters resulting from running SMIRKS-based typing on chemical topologies
-* provide an interface for exporting these parameters to MD engines (i.e. creating `openmm.Force`s)
+There is no formal specification for a `ParameterHandler` as processed by the OpenFF Toolkit, but there are some guidelines that are expected to be followed. A parameter handler requires both a [`ParameterHandler`] class, which performs parametrization, and a [`ParameterType`] class, which stores the corresponding data. For more information, see the [Toolkit documentation](openff.toolkit:users/developing.md#modular-design-features).
 
-## Creating custom `ParameterHandler`s
+The high-level objectives of a parameter handler are to:
 
-There is no formal specification for the `ParameterHandler`s as processed by the OpenFF Toolkit, but there are some guidelines that are expected to be followed.
+* store force field parameters
+* assign parameters via SMIRKS
+* enable construction and modification of parameters via the Python API
+* enable serialization and deserialization of parameters
 
 * The handler class _should_
-  * inherit from `ParameterHandler`
-  * be available in the `openff.toolkit.plugins.handlers` entry point group
-  * contain a class that inherits from `ParameterType` (a "type class")
-  * define a class attribute `_TAGNAME` that is used for serialization and handy lookups
-  * define a class attribute `_INFOTYPE` that defines the "type class"
+  * inherit from [`ParameterHandler`]
+  * be available in the `openff.toolkit.plugins.handlers` [entry point group]
+  * contain a class that inherits from [`ParameterType`] (a "type class")
+  * define a class attribute `_TAGNAME: str` that is used for serialization and handy lookups
+  * define a class attribute `_INFOTYPE` that specifies the handler's "type class"
```

Comment:
```suggestion
  * define a class attribute `_INFOTYPE: Type[ParameterType]` that specifies the handler's "type class"
```
might be too much? not sure if this is even the correct annotation ... I hadn't considered using type annotations in-line in prose before seeing you do this!

Reply:
I think it is too much, especially since it might not be clear to newcomers what `Type[foo]` means. I think the fact that it specifies a class rather than an instance is clear enough, especially given the examples, and also we do later define a type class... should be fine.